src/@types/global.d.ts

```diff
@@ -3,6 +3,7 @@ declare module '*.woff';
 declare module '*.eot';
 declare module '*.ttf';
 declare module '*.png';
+declare module '.*.jpg';
```

Comment:
Please use the same pattern as for other file types: `*.jpg`

Reply:
Oops, typo :+1: